tests/mongo-go-driver.yml

```diff
@@ -23,6 +23,7 @@ results:
       - go.mongodb.org/mongo-driver/examples/documentation_examples/TestDocumentationExamples/WithTransactionExample
       - go.mongodb.org/mongo-driver/examples/documentation_examples/TestTransactionExamples
       - go.mongodb.org/mongo-driver/mongo/TestConvenientTransactions
+      - regexp: "skipping due to environmental constraints:.*"
```

Comment:
Let's call it `output_regex` – we might want to add a regexes for test names in the future

(and let's call it regex without p because plural "regexpes" is weirder 😆)